---
title: Tools
---
# Tools

### empiricIST: a software that allows for accurate estimation of selection coefficients and credibility intervals from bulk competitions

[empiricIST](https://github.com/Matu2083/empiricIST) is an integrative framework for the analysis of bulk competition data, and includes separate programs for processing raw sequence data and correcting for sequencing errors, obtaining statistically meaningful estimates of selection coefficients in a fast and efficient manner, and for providing ready-to-use summary statistics of the MCMC analysis and its associated parameter estimates.

[empiricIST GitHub repository](https://github.com/Matu2083/empiricIST){:.l-big-button}

Also check out our manuscript on [The fitness landscape of the codon space across environments](https://doi.org/10.1101/252395) to see an example of how empiricIST can be used. We used the selection coefficients estimates to quantify changes in the topology and the topography of the codon fitness landscape in different environments considering non-synonymous and synonymous mutations.

---

### Interactive online tool to visualize the loss of a neutral Dobzhansky-Muller incompatibility in the presence of gene flow

Go directly to the tool by clicking below:

[Interactive DMI web app](https://evoldynamics.gitlab.io/dmi-js/model.html){:.l-big-button}

---

### Interactive online tool for experimental design of high-throughput bulk competitions

Go directly to the tool by clicking below:

[Optimize high-throughput bulk competitions](https://banklab.github.io/mut_scanning/visualize_page.html){:.l-big-button}

Or download and read the corresponding manuscript first: [A statistical guide to the design of deep mutational scanning experiments](http://www.genetics.org/content/early/2016/07/11/genetics.116.190462) – also available as [free preprint](http://biorxiv.org/content/early/2016/06/29/048892).

---

### Simulating and inferring selection

Exercise module for the [2018 Workshop on Population and Speciation Genomics in Český Krumlov](http://evomics.org/workshops/workshop-on-population-and-speciation-genomics/2018-workshop-on-population-and-speciation-genomics-cesky-krumlov/).

Find the R notebook here:

[simulating and inferring selection](https://evoldyn.gitlab.io/evomics-2018/){:.l-big-button}

Our [GitLab repository](https://gitlab.com/evoldyn/evomics-2018) contains the RStudio files and all other files necessary to get started with the exercise and comes with precompiled binaries for macOS and Linux and a setup script.

For more information and a pdf of the accompanying lecture see the [website of the workshop](http.evomics.org/workshops/workshop-on-population-and-speciation-genomics/2018-workshop-on-population-and-speciation-genomics-cesky-krumlov/) or download the lecture slides here: [Positive and negative selection [PDF, 6 MB]](https://evoldyn.gitlab.io/evomics-2018/lecture/positive_and_negative_selection.pdf).
