const publications = '---\nlayout: default\ntitle: Publications | Theoretical Ecology and Evolution\npermalink: /publications/\n---\n# Publications\n\nPlease [email](/people/) us for pdf copies of lab papers hidden behind a paywall.\n\nThis list shows publications of lab members during their time in the lab. For additional publications of individual lab members, see their personal pages.\n\n<sup>*</sup>equal contribution; <sup>#</sup>co-corresponding authors\n\n\n<div class="layout-publications" markdown="1">\n\n\n## 2025\n\nM. Joshi, P. Duchen, <u>A. Kapopoulou</u>, and S. Laurent. **Natural selection in Transcription Factor (TF)-DNA interaction motifs – A comparative and population genomics perspective**. Genome Biology and Evolution, doi: 10.1093/gbe/evaf212. [link](https://doi.org/10.1093/gbe/evaf212)\n\n<u>C. Chaparro-Pedraza</u>, and <u>C. Bank</u>. **Evolving life-history traits promote biodiversity via eco-evolutionary feedback mechanisms**. Plos Biology, doi: 10.1371/journal.pbio.3003492. [link](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003492) or [preprint](https://www.biorxiv.org/content/10.1101/2025.03.25.645227v1)\n\n<u>L. Marrec</u>, and <u>C. Bank</u>. **Drivers of diversity within and between microbial communities during stochastic assembly.** Journal of the Royal Society Interface, doi: 10.1098/rsif.2025.0329. [link](https://royalsocietypublishing.org/doi/full/10.1098/rsif.2025.0329) or [preprint](https://www.biorxiv.org/content/10.1101/2024.11.19.624346v1)\n\n<u>R.J. Jasper</u><sup>#</sup>, <u>A. Kapopoulou</u>, S. Peischl, B. Yilmaz, J. Becker, and <u>C. Bank</u><sup>#</sup>. **Pervasive intraspecific genetic local adaptation within the natural gut microbiome**. biorxiv, doi: 10.1101/2025.06.06.658271. [preprint](https://www.biorxiv.org/cgi/content/short/2025.06.06.658271v1).\n\nS. Conforti, <u>A. Kapopoulou</u>, <u>C. Bank</u>, B. Koechle, B. Yilmaz, and J. Becker. **Extended-spectrum beta-lactamase (ESBL)-producing E. coli in livestock and free-roaming wildlife: a combined phenotyping whole-genome sequencing One Health approach**. biorxiv, doi: 10.1101/2025.04.08.647752. [preprint](https://www.biorxiv.org/content/10.1101/2025.04.08.647752v1)\n\n<u>L. Hablützel</u>, C. Mullon, and M. Schmid. **The evolution of local adaptation in long-lived species.** Evolution, doi: 10.1093/evolut/qpaf031. [link](https://academic.oup.com/evolut/advance-article-abstract/doi/10.1093/evolut/qpaf031/8051330)\n\n<u>S.G. Das</u>, M. Mungan, and J. Krug. **Epistasis-mediated compensatory evolution in a fitness landscape with adaptational tradeoffs.** PNAS, doi: 10.1073/pnas.242252012. [link](https://www.pnas.org/doi/abs/10.1073/pnas.2422520122)\n\nN. Sharma<sup>#</sup>, <u>S.G. Das</u>, J. Krug, and A. Traulsen<sup>#</sup>. **Graph-structured populations elucidate the role of deleterious mutations in long-term evolution.** Nat. Commun. doi: 10.1038/s41467-025-57552-9. [link](https://www.nature.com/articles/s41467-025-57552-9#citeas)\n\n<u>L. Hablützel</u>, and <u>C. Bank</u>. **Studying the genetic basis of ecological interactions with intergenomic epistasis**. Oikos, doi: 10.1111/oik.10835. [link](http://doi.org/10.1111/oik.10835) or [preprint](https://ecoevorxiv.org/repository/view/6801/)\n\nK. Höcketstaller, I.A. Marti, <u>C. Bank</u>, B. Yilmaz, and J. Becker. **Antimicrobial Resistance in Wildlife across Switzerland and the Principality of Liechtenstein: Associations with Environmental Factors and Taxonomic Variation**. Environmental Research, doi: 10.1016/j.envres.2025.121968. [link](https://doi.org/10.1016/j.envres.2025.121968) or [preprint](https://www.biorxiv.org/content/10.1101/2025.01.28.635243v1)\n\nW.F. Ramirez-Duarte, B.M. Moran, D.L. Powell, <u>C. Bank</u>, V.C. Sousa, G.G. Rosenthal, M. Schumer, C.M. Rochman. **Hybridization in the Anthropocene - how pollution and climate change disrupt mate selection in freshwater fish**. Biological Reviews, doi: 10.1111/brv.13126. [link](https://onlinelibrary.wiley.com/doi/full/10.1111/brv.13126)\n\n## 2024\n\n<u>J.A. Ayala-Lopez</u>, and <u>C. Bank</u>. **What can we gain from modeling complex hybrid incompatibilities?** Evolutionary Journal of the Linnean Society, doi: 10.1093/evolinnean/kzae034. [link](https://academic.oup.com/evolinnean/advance-article/doi/10.1093/evolinnean/kzae034/7924183?utm_source=authortollfreelink&utm_campaign=evolinnean&utm_medium=email&guestAccessKey=704c8ef7-2db8-4ea4-9d99-b6198057ad2a)\n\n<u>D. McLeod</u>, <u>C. Bank</u><sup>*</sup>, and S. Gandon<sup>*</sup>. **A multilocus perspective on the evolutionary dynamics of multistrain pathogens**. PNAS, doi: 10.1073/pnas.2401578121. [link](https://www.pnas.org/doi/10.1073/pnas.2401578121) or [preprint](https://www.biorxiv.org/content/10.1101/2023.10.27.564465v1)\n\n<u>R.J. Jasper</u>, and S. Yeaman. **Local adaptation can cause both peaks and troughs in nucleotide diversity within populations.** G3, doi: 10.1093/g3journal/jkae225 [link](https://academic.oup.com/g3journal/article/14/11/jkae225/7759903)\n\n<u>M.A.D. Louro</u><sup>*#</sup>, C. Peneda<sup>*#</sup>, <u>C. Bank</u>, and M. Bettencourt-Dias<sup>#</sup>. **Adaptation of human cell populations to different levels of centriole amplification involves a two-step response**. [preprint](https://www.biorxiv.org/content/10.1101/2024.02.15.580424v2)\n\nA. Hinz, <u>A. Amado</u>, R. Kassen, <u>C. Bank</u>, and A. Wong. **Unpredictability of the fitness effects of antimicrobial resistance mutations across environments in Escherichia coli**. Molecular Biology and Evolution, doi: 10.1093/molbev/msae086. [link](https://doi.org/10.1093/molbev/msae086) or [preprint](https://www.biorxiv.org/content/10.1101/2023.10.25.563906v1)\n\n<u>A. Amado</u>, and <u>C. Bank</u>. **Ecological tradeoffs lead to complex evolutionary trajectories and sustained diversity on dynamic fitness landscapes**. Journal of Physics A, doi: 10.1088/1751-8121/ad30d0. [link](https://iopscience.iop.org/article/10.1088/1751-8121/ad30d0/meta) or [preprint](https://www.biorxiv.org/content/10.1101/2023.10.11.561986v1.abstract)\n\n<u>A.-H. Ghenu</u><sup>*#</sup>, <u>L. Marrec</u><sup>*#</sup>, and <u>C. Bank</u>. **Challenges and pitfalls of inferring microbial growth rates from lab cultures.** Frontiers in Ecology and Evolution, doi: 10.3389/fevo.2023.1313500. [link](https://www.frontiersin.org/articles/10.3389/fevo.2023.1313500/full?&utm_source=Email_to_authors_&utm_medium=Email&utm_content=T1_11.5e1_author&utm_campaign=Email_publication&field=&journalName=Frontiers_in_Ecology_and_Evolution&id=1313500) or [preprint](https://www.biorxiv.org/content/10.1101/2022.06.24.497412)\n\n## 2023\n\n<u>L. Marrec</u>. **Quantifying the impact of genotype-dependent gene flow on mutation fixation in subdivided populations.** [preprint](https://www.biorxiv.org/content/10.1101/2023.11.29.569213v1.abstract)\n\nY.-P. Lin, H.-W. Chen, P.-M. Yeh, S.S. Anand, J. Lin, <u>J. Li</u>, T. Noble, R. Nair, R. Schafleitner, M. Samsononova, E. Bishop-von-Wettberg, S. Nuzhdin, C.-T. Ting, R.J. Lawn, C.-R. Lee. **Demographic history and distinct selection signatures of two domestication genes in mungbean.** Plant Physiology, doi: 10.1093/plphys/kiad356. [link](https://academic.oup.com/plphys/article/193/2/1197/7202204)\n\n<u>J. Li</u>, and <u>C. Bank</u>. **Dominance and multi-locus interaction**. Trends in Genetics, doi: 10.1016/j.tig.2023.12.003. [link](https://www.cell.com/trends/genetics/abstract/S0168-9525(23)00262-7) or [preprint](https://arxiv.org/abs/2310.04730)\n\n<u>L. Marrec</u>, <u>C. Bank</u>, and T. Bertrand. **Solving the stochastic dynamics of population growth.** Ecology and Evolution, doi: 10.1002/ece3.10295. [link](http://dx.doi.org/10.1002/ece3.10295) or [preprint](https://www.biorxiv.org/content/10.1101/2022.11.15.516663v1)\n\nD. Grbin, S. Geček, A. Miljanović, D. Pavić, S. Hudina, J. Žučko, <u>J. Rieder</u>, S.R.R. Pisano, I. Adrian-Kalchhauser, A. Bielen. **Comparison of exoskeleton microbial communities of co-occurring native and invasive crayfish species.** Journal of Invertebrate Pathology, doi: j.jip.2023.107996. [link](https://www.sciencedirect.com/science/article/pii/S0022201123001131)\n\n<u>A. Blanckaert</u>, <u>V. Sriram</u>, and <u>C. Bank</u>. **In search of the Goldilocks zone for hybrid speciation II: hard times for hybrid speciation?** Evolution, doi: 10.1093/evolut/qpad125. [link](https://doi.org/10.1093/evolut/qpad125) or [preprint](https://www.biorxiv.org/content/10.1101/2023.02.15.528680v1)\n\n<u>L. Marrec</u>, and <u>C. Bank</u>. **Evolutionary rescue in a fluctuating environment: periodic versus quasi-periodic environmental changes.** Proceedings of the Royal Society B, doi: 10.1098/rspb.2023.0770. [link](https://royalsocietypublishing.org/doi/10.1098/rspb.2023.0770) or [preprint](https://www.biorxiv.org/content/10.1101/2022.11.03.515107v1)\n\n<u>A. Amado</u><sup>*#</sup>, <u>J. Li</u><sup>*#</sup>, and <u>C. Bank</u>. **STUN: forward-time Simulation on TUnable fitNess landscapes in recombining populations.** Bioinformatics Advances, doi:10.1093/bioadv/vbad164. [link](https://academic.oup.com/bioinformaticsadvances/article/3/1/vbad164/7424707?searchresult=1) or [preprint](https://www.biorxiv.org/content/10.1101/2023.02.15.528681v1)\n\n<u>J. Li</u><sup>*#</sup>, <u>A. Amado</u><sup>*#</sup>, and <u>C. Bank</u>. **Rapid adaptation of recombining populations on tunable fitness landscapes.** Molecular Ecology, doi: 10.1111/mec.16900. [link](http://doi.org/10.1111/mec.16900) or [preprint](https://www.biorxiv.org/content/10.1101/2022.11.16.516770v1)\n\n<u>A.-H. Ghenu</u>, <u>A. Amado</u>, I. Gordo, and <u>C. Bank</u>. **Epistasis decreases with increasing antibiotic pressure but not temperature.** Philosophical Transactions B, doi: 10.1098/rstb.2022.0058. [link](https://royalsocietypublishing.org/doi/full/10.1098/rstb.2022.0058) or [preprint](https://www.biorxiv.org/content/10.1101/2022.09.01.506172)\n\n<u>J. Li</u>, and C.-R. Lee. **The role of gene presence–absence variations on genetic incompatibility in Asian rice**. New Phytologist, doi: 10.1111/nph.18969. [link](https://nph.onlinelibrary.wiley.com/doi/full/10.1111/nph.18969?casa_token=rNrI2cnKaDkAAAAA%3AwIcO6Xr7TykwDkjFM8xNTlNDB6LZepDvc2_I9zMjrZIc2EMPkz2p9laViCr41ikHdteG7Fm2t1z4lad9)\n\n<u>J. Rieder</u>, <u>A. Kapopoulou</u>, <u>C. Bank</u>, and I. Adrian-Kalchhauser. **Metagenomics and metabarcoding experimental choices and their impact on microbial community characterization in freshwater recirculating aquaculture systems.** Environmental Microbiome, doi: 10.1186/s40793-023-00459-z. [link](https://rdcu.be/c5D08) or [preprint](https://www.biorxiv.org/content/10.1101/2022.07.05.498813v1.abstract)\n\nM.T. Wortel, D. Agashe, S.F. Bailey, <u>C. Bank</u>, K. Bisschop, T. Blankers, J. Cairns, E.S. Colizzi, <u>D. Cusseddu</u>, M.M. Desai, B. van Dijk, M. Egas, J. Ellers, A.T. Groot, D.G. Heckel, M.L. Johnson, K. Kraaijeveld, J. Krug, L. Laan, M. Lässig, P.A. Lind, J. Meijer, L.M. Noble, S. Okasha, P.B. Rainey, D.E. Rozen, S. Shitut, S.J. Tans, O. Tenaillon, H. Teotónio, J.A.G.M. de Visser, M.E. Visser, R.M.A. Vroomans, G.D.A. Werner, B. Wertheim, and P.S. Pennings. **Towards evolutionary predictions: Current promises and challenges.** Evolutionary Applications, doi: 10.1111/eva.13513. [link](https://onlinelibrary.wiley.com/doi/10.1111/eva.13513) or [earlier preprint](https://ecoevorxiv.org/4u3mg/)\n\n## 2022\n\n<u>C. Bank</u>. **Epistasis and adaptation on fitness landscapes.** Annual Review in Ecology, Evolution, and Systematics, doi: 10.1146/annurev-ecolsys-102320-112153. [link](http://www.annualreviews.org/eprint/S7XY5J79I8FDVF9PEF6K/full/10.1146/annurev-ecolsys-102320-112153) or [preprint](https://arxiv.org/abs/2204.13321)\n\n<u>J. Rieder</u>, P. M. Martin-Sanchez, O. A. Osman, I. Adrian-Kalchhauser, A. Eiler. **Detecting aquatic pathogens with field-compatible dried qPCR assays.** Journal of Microbiological Methods, doi: 10.1016/j.mimet.2022.106594. [link](https://doi.org/10.1016/j.mimet.2022.106594)\n\n<u>C. Bank</u>, <u>M.A. Schmitz</u>, and <u>A.Y. Morales-Arce</u>. **Evolutionary models predict potential mechanisms of escape from mutational meltdown.** Frontiers in Virology, doi: 10.3389/fviro.2022.886655. [link](https://www.frontiersin.org/articles/10.3389/fviro.2022.886655/full?&utm_source=Email_to_authors_&utm_medium=Email&utm_content=T1_11.5e1_author&utm_campaign=Email_publication&field=&journalName=Frontiers_in_Virology&id=886655) or [preprint](https://www.biorxiv.org/content/10.1101/2022.06.21.496937)\n\n<u>L. Lansch-Justen</u>, <u>D. Cusseddu</u>, <u>M.A. Schmitz</u>, and  <u>C. Bank</u>. **The extinction time under mutational meltdown driven by high mutation rates.** Ecology and Evolution, doi: 10.1002/ece3.9046. [link](http://dx.doi.org/10.1002/ece3.9046) or [preprint](https://www.biorxiv.org/content/10.1101/2022.02.01.478601)\n\n<u>D. Stajic</u><sup>#</sup>, <u>C. Bank</u>, and I. Gordo<sup>#</sup>. **Adaptive Potential of Epigenetic Switching During Adaptation to Fluctuating Environments.** Genome Biology and Evolution, doi: 10.1093/gbe/evac065. [link](https://academic.oup.com/gbe/article/14/5/evac065/6586088?login=true) or [preprint](https://www.biorxiv.org/content/10.1101/2021.03.11.434930v1.abstract)\n\n<u>D.V. McLeod</u>, S. Gandon. **Effects of epistasis and recombination between vaccine-escape and virulence alleles on the dynamics of pathogen adaptation.** Nature Ecology & Evolution, doi: 10.1038/s41559-022-01709-y. [link](https://doi.org/10.1038/s41559-022-01709-y)\n\n<u>J. Li</u><sup>#</sup>, M. Schumer, and <u>C. Bank</u><sup>#</sup>. **Imbalanced segregation of recombinant haplotypes in hybrid populations reveals inter- and intrachromosomal Dobzhansky-Muller incompatibilities.** Plos Genetics, doi: 10.1371/journal.pgen.1010120. [link](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1010120) or [preprint](https://www.biorxiv.org/content/10.1101/2021.08.04.454891v1)\n\n## 2021\n\n<u>M.A.D. Louro</u>, M. Bettencourt-Dias, and Jorge Carneiro. **A first-takes-all model of centriole copy number control based on cartwheel elongation.** Plos Computational Biology, doi: 10.1371/journal.pcbi.1008359. [link](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008359)\n\nM.S. Joshi, <u>A. Kapopoulou</u>, and S. Laurent. **Impact of genetic variation in gene regulatory sequences: a population genomics perspective.** Frontiers in Genetics, doi: 10.3389/fgene.2021.660899. [link](https://doi.org/10.3389/fgene.2021.660899)\n\n<u>M.A.D. Louro</u>, M. Bettencourt-Dias, and <u>C. Bank</u>. **Patterns of selection against centrosome amplification in human cell lines.** Plos Computational Biology, doi: 10.1371/journal.pcbi.1008765. [link](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008765) or [preprint](https://doi.org/10.1101/2020.01.24.918615)\n\nR.K. Butlin, M.R. Servedio, C.M. Smadja, <u>C. Bank</u>,  N.H. Barton,  S.M. Flaxman, T. Giraud, R. Hopkins, E.L. Larson, M.E. Maan, J. Meier, R. Merrill, M.A.F. Noor, D. Ortiz‐Barrientos, A. Qvarnström. **Homage to Felsenstein 1981, or why are there so few/many species?** Evolution, doi: 10.1111/evo.14235. [link](https://doi.org/10.1111/evo.14235)\n\n<u>E. Berdan</u><sup>*#</sup>, <u>A. Blanckaert</u><sup>*#</sup>, R.K. Butlin, and <u>C. Bank</u>. **Deleterious mutation accumulation and the long-term fate of chromosomal inversions.** Plos Genetics, doi: 10.1371/journal.pgen.1009411. [link](https://doi.org/10.1371/journal.pgen.1009411) or [preprint](https://www.biorxiv.org/content/10.1101/606012v1)\n\nP.A. Cote-Hammarlof<sup>*</sup>, <u>I. Fragata</u><sup>*</sup>, J.M. Flynn, K.B. Zeldovich, <u>C. Bank</u><sup>#</sup>, and D.N.A. Bolon<sup>#</sup>. **The Adaptive Potential of the Middle Domain of Yeast Hsp90.** Molecular Biology and Evolution, doi: 10.1093/molbev/msaa211. [link](https://doi.org/10.1093/molbev/msaa211) or [preprint](https://www.biorxiv.org/content/10.1101/832022v1)\n\n## 2020\n\n<u>C. Bank</u>. **Defining Theories in Evolution.** Trends in Ecology & Evolution, doi: 10.1016/j.tree.2020.05.009. (A book review of [The Theory of Evolution: Principles, Concepts, and Assumptions.](https://press.uchicago.edu/ucp/books/book/chicago/T/bo45713136.html)) [link](https://doi.org/10.1016/j.tree.2020.05.009) or [pdf](/assets/download/Defining_theories_in_evolution.pdf)\n\n<u>C. Bank</u>. **Reconciling the upsides and downsides of migration for evolutionary rescue.** Peer Community in Evolutionary Biology, doi: 10.24072/pci.evolbiol.100098. (A preprint recommendation of [Tomasini et al., 2020, Evolution](https://doi.org/10.1111/evo.14038).) [link](https://doi.org/10.24072/pci.evolbiol.100098)\n\n<u>D. Stajic</u>, and <u>C. Bank</u>. **Phenotypic switching and its evolutionary consequences**. Book Chapter in [Phenotypic Switching: Implications in Biology and Medicine](https://www.sciencedirect.com/book/9780128179963/phenotypic-switching). Preprint version: **Evolutionary consequences of epigenetically induced phenotypic switching**, EcoEvoRxiv, doi: 10.32942/osf.io/6yf4u. [preprint](https://ecoevorxiv.org/6yf4u/)\n\n<u>A. Blanckaert</u>, <u>C. Bank</u>, and J. Hermisson. **The limits to parapatric speciation III: Fast reproductive isolation in the presence of gene flow despite limited ecological differentiation.** Philosophical Transactions of The Royal Society B, doi: 10.1098/rstb.2019.0532. [link](https://doi.org/10.1098/rstb.2019.0532)\n\nJ.M. Flynn, A. Rossouw, P.A. Cote-Hammarlof, <u>I. Fragata</u>, D. Mavor, C. Hollins III, <u>C. Bank</u>, and D.N.A. Bolon. **Comprehensive fitness maps of Hsp90 show widespread environmental dependence.** eLife, doi: 10.7554/eLife.53810. [link](https://doi.org/10.7554/eLife.53810) or [preprint](https://www.biorxiv.org/content/10.1101/823468v2)\n\nR.S. Ramiro<sup>#</sup>, P. Durão, <u>C. Bank</u>, and I. Gordo<sup>#</sup>. **Low mutational load allows for high mutation rate variation in gut commensal bacteria.** Plos Biology, doi: 10.1371/journal.pbio.3000617. [link](https://doi.org/10.1371/journal.pbio.3000617) or [preprint](https://doi.org/10.1101/568709)\n\n## 2019\n\nP. Nouhaud, <u>A. Blanckaert</u>, <u>C. Bank</u>, and J. Kulmuni. **Understanding admixture: haplodiploidy to the rescue.** Trends in Ecology and Evolution, doi: 10.1016/j.tree.2019.08.013. [link](https://doi.org/10.1016/j.tree.2019.08.013)\n\nA.E. Hall, K. Kakare, V.S. Cooper, <u>C. Bank</u>, T.F. Cooper<sup>#</sup>, and F.B.G. Moore<sup>#</sup>. **Environment changes epistasis to alter trade-offs along alternative evolutionary paths.** Evolution, doi: 10.1111/evo.13825. [link](https://onlinelibrary.wiley.com/doi/10.1111/evo.13825)\n\nS. Ramos, A.R. Carlos, B. Sundaram, V. Jeney, A. Ribeiro, R. Gozzelino, <u>C. Bank</u>, E. Gjini, F. Braza, R. Martins, T.W. Ademolue, B. Blankenhaus, Z. Licínia, P. Faisca, D. Trujillo, S. Cardoso, S. Rebelo, L. del Barrio, A. Zarjou, S. Bolisetty, A. Agarwal, and M.P. Soares. **Renal control of disease tolerance to malaria.** Proceedings of the National Academy of Sciences, doi: 10.1073/pnas.1822024116. [link](https://doi.org/10.1073/pnas.1822024116)\n\n<u>I. Fragata</u>, <u>A. Blanckaert</u>, <u>M.A.D. Louro</u>, D.A. Liberles, and <u>C. Bank</u>. **Evolution in the light of fitness landscape theory.** Trends in Ecology and Evolution, doi: 10.1016/j.tree.2018.10.009. [link](https://doi.org/10.1016/j.tree.2018.10.009) or [preprint](https://www.dropbox.com/s/wiy9doprw3nhoe2/tree_ms.pdf?dl=0)\n\n## 2018\n\n<u>A. Blanckaert</u>, and <u>C. Bank</u>. **In search of the Goldilocks zone for hybrid speciation.** Plos Genetics, doi: 10.1371/journal.pgen.1007613. [link](https://doi.org/10.1371/journal.pgen.1007613) or [preprint](https://doi.org/10.1101/266254)\n\nJ.A.G.M. de Visser<sup>#</sup>, S.F. Elena<sup>#</sup>, <u>I. Fragata</u><sup>#</sup> and S. Matuszewski<sup>#</sup>. **The utility of fitness landscapes and big data for predicting evolution.** Heredity, doi: 10.1038/s41437-018-0128-4. [link](https://doi.org/10.1038/s41437-018-0128-4)\n\n<u>I. Fragata</u>, P. Simôes, M. Matos, E. Szathmáry, and M. Santos. **Playing evolution in the laboratory: from the first major evolutionary transition to global warming.** EPL (Europhysics Letters), doi: 10.1209/0295-5075/122/38001. [link](https://doi.org/10.1209/0295-5075/122/38001)\n\n<u>I. Fragata</u>, and <u>C. Bank</u>. **Let’s move beyond costs of resistance!** Peer Community In Evolutionary Biology, 100052 doi: 10.24072/pci.evolbiol.100052. (A preprint recommendation of [Lenormand et al., 2018, bioRxiv](https://doi.org/10.1101/276675).) [link](http://dx.doi.org/10.24072/pci.evolbiol.100052)\n\n<u>A. Blanckaert</u>, and J. Hermisson. **The limits to parapatric speciation II: strengthening a preexisting genetic barrier to gene flow in parapatry.** Genetics, doi: 10.1534/genetics.117.300652. [link](https://doi.org/10.1534/genetics.117.300652) or [preprint](https://doi.org/10.1101/266098)\n\n<u>I. Fragata</u><sup>*#</sup>, S. Matuszewski<sup>*#</sup>, <u>M.A. Schmitz</u>, T. Bataillon, J.D. Jensen, and <u>C. Bank</u>. **The fitness landscape of the codon space across environments.** Heredity, doi: 10.1038/s41437-018-0125-7. [link](http://dx.doi.org/10.1038/s41437-018-0125-7) or [preprint](https://doi.org/10.1101/252395)\n\n<u>A.-H. Ghenu</u><sup>*</sup>, <u>A. Blanckaert</u><sup>*</sup>, R.K. Butlin, J. Kulmuni, and <u>C. Bank</u>. **Conflict between heterozygote advantage and hybrid incompatibility in haplodiploids (and sex chromosomes).** Molecular Ecology, doi: 10.1111/mec.14482. [link](http://onlinelibrary.wiley.com/doi/10.1111/mec.14482/abstract) or [preprint](https://doi.org/10.1101/196469) \n\n## 2017\n\n<u>I. Fragata</u>, and <u>C. Bank</u>. **What doesn’t kill us makes us stronger: can Fisher’s Geometric model predict antibiotic resistance evolution?** Peer Community in Evolutionary Biology, doi: 10.24072/pci.evolbiol.100028. (A post-print recommendation of [Harmand et al., 2016, Evolution](https://doi.org/10.1111/evo.13111).) [link](https://doi.org/10.24072/pci.evolbiol.100028)\n\nL. Ormond, P. Liu, S. Matuszewski, N. Renzette, <u>C. Bank</u>, K.B. Zeldovich, D.N.A. Bolon, T.F. Kowalik, R.W. Finberg, J.D. Jensen<sup>#</sup>, and J.P. Wang<sup>#</sup>. **The combined effect of oseltamivir and favipiravir on influenza A virus evolution.** Genome Biology and Evolution, doi: 10.1093/gbe/evx138. [link](https://doi.org/10.1093/gbe/evx138)\n\nS. Matuszewski, L. Ormond, <u>C. Bank</u>, and J.D. Jensen. **Two sides of the same coin: a population genetics perspective on lethal mutagenesis and mutational meltdown.** Virus Evolution, doi: 10.1093/ve/vex004. [link](https://doi.org/10.1093/ve/vex004)\n\n## 2016\n\n<u>C. Bank</u><sup>*</sup>, S. Matuszewski<sup>*</sup>, R.T. Hietpas, and J.D. Jensen. **On the (un)predictability of a large intragenic fitness landscape.** Proceedings of the National Academy of Sciences, doi: 10.1073/pnas.1612676113. [link](https://doi.org/10.1073/pnas.1612676113) or [preprint](https://doi.org/10.1101/048769)\n\n<u>C. Bank</u>, N. Renzette, P. Liu, S. Matuszewski, H. Shim, M. Foll, D.N.A. Bolon, K.B. Zeldovich, T.F. Kowalik, R.W. Finberg, J.P. Wang<sup>#</sup>, and J.D. Jensen<sup>#</sup>. **An experimental evaluation of drug-induced mutational meltdown as an antiviral treatment strategy.** Evolution, doi: 10.1111/evo.13041. [link](https://doi.org/10.1111/evo.13041) or [preprint](https://doi.org/10.1101/048934)\n\nK.K. Irwin, S. Laurent, S. Matuszewski, S. Vuilleumier, L. Ormond, H. Shim, <u>C. Bank</u>, and J.D. Jensen. **On the importance of skewed offspring distributions and background selection in viral population genetics.** Heredity, doi: 10.1038/hdy.2016.58 [link](https://doi.org/10.1038/hdy.2016.58) or [preprint](https://doi.org/10.1101/048975). Read our summary of this article [here](https://evoldynamics.org/2016/12/05/on-the-importance-of-skewed-offspring-distributions-and-background-selection-in-viral-population-genetics/).\n\nS. Matuszewski<sup>*#</sup>, M.E. Hildebrandt<sup>*</sup>, <u>A.-H. Ghenu</u>, J.D. Jensen, and <u>C. Bank</u><sup>#</sup>. **A statistical guide to the design of deep mutational scanning experiments.** Genetics, doi: 10.1534/genetics.116.190462. [link](https://doi.org/10.1534/genetics.116.190462) or [preprint](https://doi.org/10.1101/048892)\n\nL. Jiang, P. Liu, <u>C. Bank</u>, N. Renzette, K. Prachanronarong, L.S. Yilmaz, D.R. Caffrey, K.B. Zeldovich, C.A. Schiffer, T.F. Kowalik, J.D. Jensen, R.W. Finberg, J.P. Wang<sup>#</sup>, and D.N.A. Bolon<sup>#</sup>. **A balance between inhibitor binding and substrate processing confers influenza drug resistance.** Journal of Molecular Biology, doi: 10.1016/j.jmb.2015.11.027. [link](https://doi.org/10.1016/j.jmb.2015.11.027)\n\n## 2015\n\n<u>C. Bank</u>, R.T. Hietpas, J.D. Jensen, and D.N.A. Bolon. **A systematic survey of an intragenic epistatic landscape.** Molecular Biology and Evolution, doi: 10.1093/molbev/msu301. [link](https://doi.org/10.1093/molbev/msu301) or [preprint](https://doi.org/10.1101/010645)\n\n## 2014\n\n<u>C. Bank</u>, G. Ewing, A. Ferrer-Admetlla, M. Foll, and J.D. Jensen. **Thinking too positive? Revisiting current methods of population-genetic selection inference.** Trends in Genetics, doi: 10.1016/j.tig.2014.09.010. [link](https://doi.org/10.1016/j.tig.2014.09.010) or [preprint](https://doi.org/10.1101/009654)\n\n<u>C. Bank</u>, R.T. Hietpas, A. Wong, D.N.A. Bolon<sup>#</sup>, and J.D. Jensen<sup>#</sup>. **A Bayesian MCMC approach to assess the complete distribution of fitness effects of new mutations: uncovering the potential for adaptive walks in challenging environments.** Genetics, doi: 10.1534/genetics.113.156190. [link](https://doi.org/10.1534/genetics.113.156190)\n\nM. Foll<sup>*</sup>, Y.-P. Poh<sup>*</sup>, N. Renzette, A. Ferrer-Admetlla, <u>C. Bank</u>, H. Shim, A.-S. Malaspinas, G. Ewing, D. Wegmann, D.R. Caffrey, K.B. Zeldovich, D.N. Bolon, J. Wang, T.F. Kowalik, C.A. Schiffer, R.W. Finberg, and J.D. Jensen. **Influenza virus drug resistance: a time-sampled population-genetics perspective.** PLOS Genetics, doi: 10.1371/journal.pgen.1004185. [link](https://doi.org/10.1371/journal.pgen.1004185)\n\nO. Seehausen, R.K. Butlin, I. Keller, C. Wagner, J. Boughman, P. Hohenlohe, C. Peichel, G.-P. Saetre, <u>C. Bank</u>, A. Brännström, A. Brelsford, C. Clarkson, F. Eroukhmanoff, J.L. Feder, M.C. Fischer, A.D. Foote, P. Franchini, C.D. Jiggins, F.C. Jones, A.K. Lindholm, K. Lucek, M.E. Maan, D.A. Marques, S.H. Martin, B. Matthews, J.I. Meier, M. Möst, M.W. Nachman, E. Nonaka, C.L. Peichel, D.J. Rennison, G.-P. Saetre, J. Schwarzer, C.E. Wagner, E.T. Watson, A.M. Westram, and A. Widmer. **Genomics and the origin of species.** Nature Reviews Genetics, doi: 10.1038/nrg3644. [link](https://doi.org/10.1038/nrg3644)\n\n## 2013\n\nR.T. Hietpas<sup>*</sup>, <u>C. Bank</u><sup>*</sup>, J.D. Jensen<sup>#</sup>, and D.N.A. Bolon<sup>#</sup>. **Shifting fitness landscapes in response to altered environments.** Evolution, doi: 10.1111/evo.12207. [link](https://doi.org/10.1111/evo.12207)\n\n## 2012\n\n<u>C. Bank</u>, R. Bürger, and J. Hermisson. **The limits to parapatric speciation: Dobzhansky-Muller incompatibilities in a continent-island model.** Genetics, doi: 10.1534/genetics.111.137513. [link](https://doi.org/10.1534/genetics.111.137513). \n\n<u>C. Bank</u>, J. Hermisson, and M. Kirkpatrick. **Can reinforcement complete speciation?** Evolution, doi: 10.1111/j.1558-5646.2011.01423.x. [link](https://onlinelibrary.wiley.com/doi/full/10.1111/j.1558-5646.2011.01423.x)\n*This paper was listed in a Perspective on [Recent work in speciation research by women authors](https://onlinelibrary.wiley.com/doi/10.1111/evo.14444) in the journal Evolution, which appeared in 2022.*\n\n</div>\n';
export {
  publications as default
};
